.github/workflows/ios.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     runs-on: macOS-latest
     steps:
       - uses: actions/checkout@v1
+      - name: List available versions of Xcode
```

Comment:
not sure if we need that step still

Reply:
Not really, but I'm tempted to keep it since it takes virtually no time, and it might hint at upcoming changes when xocde 12.5 turns up on github actions, as it looks like this will cause a bunch of upgrade issues once more :(

Reply:
okay, so let's keep this line then.